tests/utils.py

```diff
@@ -254,6 +270,27 @@ class TranspileTestCase(TestCase):
     def setUp(self):
         setUpSuite()
 
+    def setUpClass():
+        global _jvm
+        global _jvm_configured
```

Comment:
Do we need both `_jvm` and `_jvm_configured`? Couldn't we just use `_jvm is None` as as marker for "we have no JVM", and when you invoke `runAsJava()`, start a new JVM if one doesn't exist? That would also avoid duplicating the `Popen` logic.